extensions/amp-social-share/1.0/social-share.js

```diff
@@ -60,43 +63,36 @@ export function SocialShare(props) {
   return (
     <div
       role="button"
-      tabindex={props['tabIndex'] || '0'}
+      tabindex={tabIndex || '0'}
```

Comment:
Let's give `tabIndex` its default value when you destructure it instead of here.

Reply:
Thanks, updated!